src/MerchantCenter/MerchantStatuses.php

```diff
@@ -428,24 +428,27 @@ protected function get_product_issues( array $statuses ): array {
 				'applicable_countries' => [],
 				'source'               => 'mc',
 			];
-			foreach ( $mc_product_status->getItemLevelIssues() as $item_level_issue ) {
-				if ( 'merchant_action' !== $item_level_issue->getResolution() ) {
-					continue;
+			$item_level_issues      = $mc_product_status->getItemLevelIssues();
+			if ( ! empty( $item_level_issues ) ) {
+				foreach ( $item_level_issues as $item_level_issue ) {
```

Comment:
While the logic does stay the same, in this case I think it would be nicer to stick to the same style as the rest of the function where we either continue a for loop or return early. So we can avoid too many levels of indentation. The called functions will also be optimized when run so no need to store the return value in a separate variable.

So something like:
```php
// Confirm there are issues for this product.
if ( empty( $mc_product_status->getItemLevelIssues() ) {
	continue;
}

$product_issue_template = [
....
```